Update l10n_br.xml

bvnovaes opened the pull request:
Added localization for Brazilian Portuguese.

bvnovaes committed:
Update l10n_br.xml

loki79uk commented:
There are a few new language strings in the latest update. Would it be possible to look at those too?

bvnovaes committed:
Update l10n_br.xml

Updated the translation with the new settings.

bvnovaes committed:
Merge branch 'main' into patch-1

bvnovaes committed:
Update l10n_br.xml

Adjustment for better understanding of the term.

bvnovaes commented:
> There are a few new language strings in the latest update. Would it be possible to look at those too?

No problem! I updated the file with the new strings. Thanks for the amazing work!

bvnovaes committed:
Update l10n_br.xml

Adjust of terms to reflect better understanding of the actions of double tap and long hold.

bvnovaes committed:
Update modDesc.xml
